gnovm/cmd/gno/test.go

```diff
@@ -328,9 +328,10 @@ func gnoTestPkg(
 
 		// tfiles, ifiles := gno.ParseMemPackageTests(memPkg)
 		tfiles, ifiles := parseMemPackageTests(memPkg)
+		testPkgName := getPkgNameFromFileset(ifiles)
 
 		// run test files in pkg
-		{
+		if !strings.HasSuffix(testPkgName, "_test") {
```

Comment:
Not really sure why this condition wasn't present already...

This prevent from the `testStore` to be setted twice, it was making issues with `init()` functions from packages